README.md

```diff
@@ -68,12 +68,11 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- Using one of our [executable installers](https://github.com/spyder-ide/spyder/releases)
+- Or installing one of these scientific Python distributions:
```

Comment:
This line should read

    - Using one of these scientific Python distributions: